### DISCUSSION TITLE
［灌水］今天来点大家想看的学习资料啊😄

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#5163] [灌水]
趁着还没被举报赶紧看👀![下载](https://github.com/user-attachments/assets/5318bc61-dad3-4009-a927-86571c725fad)

**Comment by Yizakl** [comment]
难绷，你钓上两只大鱼
（门槛还是太低了）
![IMG_0105](https://github.com/user-attachments/assets/8ffc637f-41ef-4714-af53-5ab18388c10b)

**Reply by KELERSAMA** [reply]
大佬<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_4941628593528507734.png" alt="阿姬倒地了03" width="75" >

**Reply by Yizakl** [reply]
我是FW喵

**Reply by KELERSAMA** [reply]
我是fw喵嗷

**Reply by Yizakl** [reply]
喵呜？你才是大佬喵

**Reply by fzdqz** [reply]
钓鱼佬永不空军😄

**Reply by TNCNET** [reply]
az，看来图样图森破了
不过我下意识的会以为是nsfw内容（

**Comment by KiryuuAkira** [comment]
我勒个红色学习资料啊

**Comment by Finkawa** [comment]
**这真的是能举报的嘛**

**Comment by Junjun280** [comment]
哈哈哈

**Comment by lllliufeng** [comment]
说的道理

**Comment by BunnYbota** [comment]
真·学习资料

**Comment by Kazuha-hhh** [comment]
钓佬我看到你好多次了
![Uploading t018c90de84e25beeb5.jpg…]()

**Comment by SA0330** [comment]
666
![Uploading Screenshot_20240920_211514_com.tencent.mm.jpg…]()

**Comment by SA0330** [comment]
![Screenshot_20240920_211514_com tencent mm](https://github.com/user-attachments/assets/7d345d92-5d4d-41ee-854f-863d60c723a6)

**Comment by IXfromThorne** [comment]
笑死了wwwww

**Comment by wzyaeu** [comment]
最好看的了，还有红色😍😍😍😍

**Comment by error100e1** [comment]
看起来藏着黄色东西的图片藏着一大块中国红
